README.md

```diff
@@ -55,4 +55,12 @@ bin/makebag bentleyaudio <barcode> -s /path/to/content --metadata-type EAD --met
 
 ```
 bin/makebag digital barcode path_to_bag
+
+
+## Bagging Video Game Content
+
+`makebag video_game` will add chipmunk bagging information to an existing bag. The bag should follow the [UMICH disk imaging profile](https://www.umich.edu/~aelkiss/UMICH-Disk-Imaging-profile.json)
```

Comment:
This is the wrong profile. We should decide whether we publish this anywhere or just refer to the file on disk.

Reply:
not sure what profile to use.

Reply:
We added the real profile to the server repo (https://github.com/mlibrary/chipmunk/pull/112/files#diff-ef536aed197983d2176ef0b9787424477815dca573423e5d299109628590affc). We need it here, too, and can just mention or link the path to it within this repo.